sycl/plugins/hip/pi_hip.cpp

```diff
@@ -2389,6 +2390,8 @@ pi_result hip_piEnqueueMemBufferRead(pi_queue command_queue, pi_mem buffer,
 
 pi_result hip_piEventsWait(pi_uint32 num_events, const pi_event *event_list) {
 
+                 int         a;
+  std::cout << "In piEvetsWait" << std::endl;
```

Comment:
Please remove this.

Reply:
These lines are removed in the [latest commit](https://github.com/intel/llvm/pull/5880/commits/2d2621184593aac71561d73a015a733322b7d44f).

Do let me know if I'm missing somthing

Reply:
Oh, my bad. This is fine.